helm/common/utils.sh

```diff
@@ -72,3 +72,17 @@ function meets_min_version() {
     false
   fi
 }
+
+# Runs a Helm unit test using the 'helm-unittest' Helm plugin.
+# Reference: https://github.com/quintush/helm-unittest/blob/master/DOCUMENT.md
+
+# Install the 'helm-unittest' plugin if it hasn't been install already
+function run_helm-unittest() {
```

Comment:
Bash functions should use lower_snake_case (https://google.github.io/styleguide/shellguide.html#s7.1-function-names), so `run_helm_unittest` would be more consistent.